src/network/clientpackethandler.cpp

```diff
@@ -773,14 +773,15 @@ void Client::handleCommand_PlaySound(NetworkPacket* pkt)
 	v3f pos;
 	u16 object_id;
 	bool loop;
+	float fade;
 
-	*pkt >> server_id >> name >> gain >> type >> pos >> object_id >> loop;
+	*pkt >> server_id >> name >> gain >> type >> pos >> object_id >> loop >> fade;
```

Comment:
you should be retrocompatible, use a try/catch directive to handle the missing fade